src/pages/Team/Team.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useRef } from 'react';
+import { Link } from 'react-router-dom';
+import Modal from '../../components/Modal/Modal';
+import members from '../../local-json/members.json';
+import AddMember from '../../utils/AddMember/AddMember';
+import { Wrapper, TitleArea } from './Team.styles';
+
+
+const Team = () => {
+
+    const [membersData] = useState(members);
+    const modal = useRef();
```

Comment:
useState hook should be used only if you are going to modify that state in any place.
This state is not modified so it can be like const membersData = members;